src/trans.rs

```diff
@@ -145,11 +179,11 @@ impl<'v, 'tcx: 'v> BinaryenFnCtxt<'v, 'tcx> {
 
         // Translate arg and ret tys to wasm
         let binaryen_args: Vec<_> = self.sig.inputs.iter().map(|t| rust_ty_to_binaryen(t)).collect();
-        let mut needs_ret_local = false;
+        let mut needs_ret_var = false;
```

Comment:
plus it's the more correct wasm terminology (and vars called locals caused me to miss the fact that relooper helper was using the wrong local index :)